tests/config/settings-config.yaml

```diff
@@ -0,0 +1,2 @@
+server-settings:
+    server.thread_pool: 1000
```

Comment:
Yes.  I like the inline, because I can see what the test is doing without
looking at yet another file.

On Fri, Jan 22, 2016 at 3:18 PM, Roni Choudhury <notifications@github.com>
wrote:

> In tests/config/settings-config.yaml
> <https://github.com/Kitware/tangelo/pull/543#discussion_r50585703>:
>
> > @@ -0,0 +1,2 @@
> > +server-settings:
> > +    server.thread_pool: 1000
>
> Apparently not :)
>
> It was connected to the test I removed, in favor of the ones you wrote.
> Yours just create a config inline, right? If that's the case, I'll remove
> this file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/tangelo/pull/543/files#r50585703>.
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439